openfisca_aotearoa/variables/social_security/childcare_subsidy.py

```diff
@@ -0,0 +1,64 @@
+
+# -*- coding: utf-8 -*-
+
+from openfisca_core.model_api import *
+from openfisca_aotearoa.entities import Person
+
+
+class childcare_subsidy__is_a_dependant_child(Variable):
```

Comment:
http://www.legislation.govt.nz/regulation/public/2004/0268/latest/DLM282545.html
is section 12, so would be in a folder called 12